ecosystem_tests/ecosystem_tests_cli/commands/blueprint_linting.py

```diff
@@ -0,0 +1,122 @@
+########
+# Copyright (c) 2014-2022 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tempfile
+import subprocess
+from git import Repo
+from github import Github
+from ..logger import logger
+from datetime import datetime
+from ...ecosystem_tests_cli import ecosystem_tests
+from ecosystem_cicd_tools.new_cicd.github import with_github_client
+
+
+@ecosystem_tests.command(
+        name='blueprint-linting',
+        short_help='validate blueprints in a repo using cfy-lint autofix.')
+@ecosystem_tests.options.github_token
+@ecosystem_tests.options.repo
+@ecosystem_tests.options.org
+@ecosystem_tests.options.pull_request_title
+def blueprint_linting(github_token=None,
+                      repo=None,
+                      org=None,
+                      pull_request_title=None):
+
+    # prep variables
+    branch_name = time = datetime.now().strftime("%Y-%m-%d_%H-%M-%S-%f")
+    github_token = github_token or os.environ("GITHUB_TOKEN")
+    if org and repo:
+        repo_name = org + "/" + repo
+    else:
+        repo_name = (os.environ("CIRCLE_PROJECT_USERNAME") + "/" +
+                     os.environ("CIRCLE_PROJECT_REPONAME"))
+    directory = tempfile.mkdtemp(prefix=time)
+    pull_request_title = pull_request_title or "cfy-lint autofix " + time
+
+    file_type = ".yaml"
```

Comment:
This can be moved to the area below imports at the top of the file and renamed to a constant:
```suggestion
    FILE_TYPE = ".yaml"
```

However, I wonder if we do not need to check if a file is a blueprint first. This could be like opening it and seeing if it starts with `tosca_definitions_version: `.